src/main/java/com/spotinst/sdkjava/model/AuditEventLogs.java

```diff
@@ -0,0 +1,250 @@
+package com.spotinst.sdkjava.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Created by ramrutha on 13/02/2023.
+ */
+public class AuditEventLogs {
```

Comment:
Rename to AuditLogEvents
Move to C:\Github\spotinst-sdk-java\src\main\java\com\spotinst\sdkjava\model\bl\admin\account

Reply:
Done